qa/integration-tests/src/test/java/io/camunda/it/client/DecisionSearchTest.java

```diff
@@ -475,7 +477,11 @@ public void shouldValidatePagination() {
     final var resultAfter =
         camundaClient
             .newDecisionRequirementsSearchRequest()
-            .page(p -> p.searchAfter(Collections.singletonList(key)))
+            .page(
+                p ->
+                    p.searchAfter(
+                        Collections.singletonList(
+                            new PageObject().value(String.valueOf(key)).type(TypeEnum.INT64))))
```

Comment:
❓ Could we instead use the first or last sort values we received in the first response to do the `searchAfter` accordingly? That would directly validate that this fits together as well.

Reply:
Ah wait, we do this in the `ProcessDefinitionSearchTest`already, right?